declaration.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "*.scss";
+declare module "*.svg";
```

Comment:
These are both so that Typescript doesn't throw errors for these filetypes. If we ended up using more filetypes we will want to add them here. This file may eventually get pretty big if we need to make custom type declarations or file resolutions.